### PR TITLE
Add missing manifest icon type (closes #219)

### LINKED ISSUE #219: manifest.json icons should have type: "image/png" to work in Chrome
If you don't tell Chrome that the icon is of type: "image/png" then Chrome will not be able to do the Add to Homescreen correctly, specifically the app install banner will not be triggered.  See https://events.google.com/io2015/manifest.json as a good example.

For example: 
```
"icons": [{
        "src": "images/touch/icon-128x128.png",
        "sizes": "128x128"
      }, {
        "src": "images/touch/apple-touch-icon.png",
        "sizes": "152x152"
      }, {
        "src": "images/touch/ms-touch-icon-144x144-precomposed.png",
        "sizes": "144x144"
      }, {
        "src": "images/touch/chrome-touch-icon-192x192.png",
        "sizes": "192x192"
      }],
```

Should be

```
"icons": [{
        "src": "images/touch/icon-128x128.png",
        "sizes": "128x128",
        "type": "image/png"
      }, {
        "src": "images/touch/apple-touch-icon.png",
        "sizes": "152x152",
        "type": "image/png"
      }, {
        "src": "images/touch/ms-touch-icon-144x144-precomposed.png",
        "sizes": "144x144",
        "type": "image/png"
      }, {
        "src": "images/touch/chrome-touch-icon-192x192.png",
        "sizes": "192x192",
        "type": "image/png"
      }],
```

### PR DESCRIPTION
**Pull request by addyosmani**
reviewer: @paulkinlan

### CODE REVIEW

File: app/manifest.json [hunk 1]
```diff
@@ -3,16 +3,20 @@
...
         "src": "images/touch/ms-touch-icon-144x144-precomposed.png",
-        "sizes": "144x144"
+        "sizes": "144x144",
+        "type": "image/png"
       }, {
         "src": "images/touch/chrome-touch-icon-192x192.png",
-        "sizes": "192x192"
+        "sizes": "192x192",
+        "type": "image/png"
       }],
```
Comment (arthurvr) [inline app/manifest.json]: this indentation seems broken. For the rest this looks good to me.

Comment (addyosmani) [inline app/manifest.json]: Do you mean the indentation for the entire `"icons"` array? Are you suggesting something like..

```
{
  "name": "Polymer Starter Kit",
  "short_name": "Polymer Starter Kit",
  "icons": [{
        "src": "images/touch/icon-128x128.png",
        "sizes": "128x128",
        "type": "image/png"
      }, {
        "src": "images/touch/apple-touch-icon.png",
        "sizes": "152x152",
        "type": "image/png"
      }, {
        "src": "images/touch/ms-touch-icon-144x144-precomposed.png",
        "sizes": "144x144",
        "type": "image/png"
      }, {
        "src": "images/touch/chrome-touch-icon-192x192.png",
        "sizes": "192x192",
        "type": "image/png"
      }
  ],
  "start_url": "/#!/",
  "display": "standalone"
}
```

would be better?

Comment (arthurvr) [inline app/manifest.json]: Yeah the whole array looked a little weird to my (specifically lines 20-21) but it's no big. Let's just keep it as-is for the time being.

**Comment by addyosmani** [comment]
LGTY for a merge, @arthurvr?

**Comment by arthurvr** [comment]
Yepp!

**Pull request merged**